### PR TITLE
Ensure tests that run on kathmandunet also run on mondaynet(alpha)

### PR DESCRIPTION
**Pull request by roxaneletourneau**
Thank you for your contribution to Taquito.

Before submitting this PR, please make sure:

- [ ] Your code builds cleanly without any errors or warnings
- [ ] You have added unit tests
- [ ] You have added integration tests (if relevant/appropriate)
- [ ] All public methods or types have TypeDoc coverage with a complete description, and ideally an @example
- [ ] You have added or updated corresponding documentation
- [ ] If relevant, you have written a first draft summary describing the change for inclusion in Release Notes. 

## Release Note Draft Snippet

__If relevant, please write a summary of your change that will be suitable for
inclusion in the Release Notes for the next Taquito release.__

### CODE REVIEW

File: integration-tests/contract-execute-on-chain-view.spec.ts [hunk 1]
```diff
@@ -75,10 +75,10 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
         await contract.contractViews.test_failwith(3).executeView({ viewCaller: contract.address });
       } catch (error: any) {
         const protocol = (await Tezos.rpc.getProtocols()).protocol
-        if(protocol === Protocols.PtKathman) {
-          expect(error).toBeInstanceOf(ViewSimulationError)
-        } else {
+        if(protocol === Protocols.PtJakart2) {
```
Comment (michaelkernaghan) [inline integration-tests/contract-execute-on-chain-view.spec.ts:78]: Why is it changed from K to J?

**Review by michaelkernaghan — COMMENTED** [review]
looks good, just the one question

File: integration-tests/contract-execute-on-chain-view.spec.ts (hunk 1 above)
Comment (roxaneletourneau) [reply integration-tests/contract-execute-on-chain-view.spec.ts:78]: I reversed the if/else statement so Kathmandunet, mondaynet, and all incoming protocols use the else case without us having to maintain anything.

**Comment by michaelkernaghan** [comment]
all good

**Pull request merged**